Description: Add support for libffi8
Origin: other, https://github.com/cffi/cffi/pull/191/commits/06c2b5ffc5a0dcf421c80c5d366f1d34aafa0fb6
Bug: https://github.com/cffi/cffi/pull/191
Last-Update: 2021-11-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/libffi/libffi.lisp
+++ b/libffi/libffi.lisp
@@ -31,7 +31,7 @@
   (:darwin (:or "libffi.dylib" "libffi32.dylib" "/usr/lib/libffi.dylib"))
   (:solaris (:or "/usr/lib/amd64/libffi.so" "/usr/lib/libffi.so"))
   (:openbsd "libffi.so")
-  (:unix (:or "libffi.so.7" "libffi32.so.7" "libffi.so.6" "libffi32.so.6" "libffi.so.5" "libffi32.so.5"))
+  (:unix (:or "libffi.so.8" "libffi32.so.8" "libffi.so.7" "libffi32.so.7" "libffi.so.6" "libffi32.so.6" "libffi.so.5" "libffi32.so.5"))
   (:windows (:or "libffi-7.dll" "libffi-6.dll" "libffi-5.dll" "libffi.dll"))
   (t (:default "libffi")))
 
